package.json

```diff
@@ -74,6 +74,8 @@
     "import:reports:local": "./node_modules/.bin/babel-node ./src/tools/importSSActivityReports.js",
     "import:reports": "node ./build/server/tools/importSSActivityReports.js",
     "import:goals:local": "./node_modules/.bin/babel-node ./src/tools/importTTAPlanGoals.js",
+    "create:goals": "node ./build/server/tools/goalPilotCLI.js",
```

Comment:
with the introduction of the typescript compiler, the build paths have changed slightly, so this path will not work once merged. it should be updated to `./build/server/src/tools/goalPilotCLI.js`:

<img width="298" alt="Screen Shot 2023-01-25 at 11 12 08 AM" src="https://user-images.githubusercontent.com/17074357/214647694-1360dedc-ce6a-4022-b91d-d0cccd0ec795.png">

Reply:
Thanks. Adjusted.